src/test/java/teammates/test/cases/ui/InstructorCoursesPageActionTest.java

```diff
@@ -85,10 +85,10 @@ public void testExecuteAndPostProcess() throws Exception{
         
         InstructorCoursesPageData pageData = (InstructorCoursesPageData)r.data;
         assertEquals(instructorId, pageData.account.googleId);
-        assertEquals(2, pageData.allCourses.size());
-        assertEquals(0, pageData.archivedCourses.size());
-        assertEquals("", pageData.courseIdToShow);
-        assertEquals("", pageData.courseNameToShow);
+        assertEquals(2, pageData.getActiveCourses().getSize());
+        assertEquals(0, pageData.getArchivedCourses().getSize());
```

Comment:
Even if getArchivedCourses() returns 0, you should still get the sizes of both and add them together.
That should be the intent of the original test: to test the total number of courses, not just the number of active ones (even if they are the same).